R/make.R

```diff
@@ -1045,8 +1045,9 @@ full_graph <- function(...) constructor_spec(make_full_graph, ...)
 #' @param directed Whether to create a directed lattice.
 #' @param mutual Logical, if `TRUE` directed lattices will be
 #'   mutually connected.
-#' @param circular Logical, if `TRUE` the lattice or ring will be
-#'   circular.
+#' @param periodic Logical vector, Boolean vector, defines whether the generated lattice is
+#'   periodic along each dimension. This parameter may also be `NULL`, which
+#'   implies that the lattice will not be periodic.
```

Comment:
The docs need an update to match the new interface. The new interface looks good.

Reply:
@szhorvat Do you mean description in the `make.R` of this function? Is current desciption is not valid anymore?
```R
#' `make_lattice()` is a flexible function, it can create lattices of
#' arbitrary dimensions, periodic or aperiodic ones. It has two
#' forms. In the first form you only supply `dimvector`, but not
#' `length` and `dim`. In the second form you omit
#' `dimvector` and supply `length` and `dim`.
```

Reply:
Specifically the change I am commenting on is not valid anymore. It says that the parameter value has to be a logical vector or NULL, but in fact it is either a logical vector or a logical scalar, with specific interpretation for each.

Reply:
@Antonov548 was this comment solved?